test/unit/utils/cookie_test.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+// npm dependencies
+const {expect} = require('chai')
+const sinon = require('sinon')
+const proxyquire = require('proxyquire').noPreserveCache()
+
+const getCookiesUtil = sessionsStub => {
+  if (sessionsStub) return proxyquire('../../../app/utils/cookie', {'client-sessions': sessionsStub})
+  return proxyquire('../../../app/utils/cookie', {})
+}
+
+describe('cookie configuration', function () {
```

Comment:
Great that you added `cookie` util test! 👍 
May be it's good to define global const in `describe('cookie configuration'` like:
```
const defaultProcessEnv = {
  SESSION_ENCRYPTION_KEY: 'naskjwefvwei72rjkwfmjwfi72rfkjwefmjwefiuwefjkbwfiu24fmjbwfk',
  COOKIE_MAX_AGE: 10800000
}
```
and reuse it in the test?
Or even better you can use https://github.com/alphagov/pay-products-ui/blob/62a8d73fe5b19f32c6b15a2ad18cef9844c5ba04/test/test_helpers/test-env.js which reads the https://github.com/alphagov/pay-products-ui/blob/62a8d73fe5b19f32c6b15a2ad18cef9844c5ba04/test/test.env file directly with the public `naskjwefvwei72rjkwfmjwfi72rfkjwefmjwefiuwefjkbwfiu24fmjbwfk` test secret and all the properties

Reply:
These test variables are already defined in test.env file and they get loaded by mocha on startup. I would leave the values in tests where I test them, so I will make it obvious and easier  to see what we are testing.